README.md

```diff
@@ -1,77 +1,150 @@
-# Verification-System
+# Verification System
 
-![out](https://github.com/viam-labs/verification-system/assets/8298653/7af85327-8d6f-4691-ade8-bb8e651c57c8)
+This module implements Viam's machine learning (ML) model service and vision service to provide a tool for facial verification and object detection.
```

Comment:
It doesn't implement the ML model service, just can use mlmodel vision services. Module itself is a vision service: `vision / classifier:verification-system`
Change to match template like "This module implements the rdk vision API in a verification-system model."